README.md

```diff
@@ -47,8 +47,8 @@ STORY SCENES:
 		choice:
 			text: “<string>”
 			next scene: <scene name>
-			conditional: <stat name> <[==, >=, <=, >, <] for int, [==] for string> <value>	(optional)
-			stat: <stat name> <[+,-,*,/,=] for int, [=] for string> <value>	(optional)
+			conditional: "<stat name> <[==, >=, <=, >, <] for int, [==] for string> <value>" (optional)
+			stat: "<stat name> <[+,-,*,/,=] for int, [=] for string> <value>" (optional)
```

Comment:
can you give a quick run down as to why the double quotes are necessary?

Reply:
They split if the stat name has spaces

Reply:
It only happens when I try to make my own files. If I edit them, the tokenizer works as expected.

Also, are string stats only one word, or multiple?

Reply:
you can keep them in for now to be safe. we ideally want the user to do as little as possible, but we can refine this later if required